core/src/com/unciv/logic/civilization/GoldenAgeManager.kt

```diff
@@ -34,6 +37,9 @@ class GoldenAgeManager : IsPartOfGameInfoSerialization {
         turnsLeftForCurrentGoldenAge += turnsToGoldenAge.toInt()
         civInfo.addNotification("You have entered a Golden Age!", "StatIcons/Happiness")
         civInfo.popupAlerts.add(PopupAlert(AlertType.GoldenAge, ""))
+        if (civInfo.isPlayerCivilization() && (civInfo.gameInfo.currentPlayerCiv == civInfo)) {
```

Comment:
Here too, golden ages currently start at the end of your turn (see 'endTurn' below), so you'd click 'end turn' and then it would make a sound, which is strange.
This can easily be changed to have them start on the startTurn function with not problems I can think of.